simulator/modules/decode/decode.h

```diff
@@ -21,6 +21,9 @@ class Decode : public Log
     using BypassingUnit = DataBypass<FuncInstr>;
 
     private:
+        uint64  num_jumps       = 0;
+        float64 num_mispredictions = 0;
```

Comment:
Please use integer counters just for simulation speed reasons (floating addition).
We need floating point only once when we divide the numbers.